src/org/labkey/test/components/labkey/ui/samples/SampleTypeDesigner.java

```diff
@@ -208,6 +209,24 @@ public String getParentAliasSelectText(int index)
         protected final WebElement cancelButton = Locator.button("Cancel").findWhenNeeded(this);
         protected final WebElement saveButton = Locator.button("Save").findWhenNeeded(this);
 
+        // the SM app uses alternate text for the sample type designer save buttons
+        protected final WebElement finishButton = Locator.buttonContainingText("Finish").findWhenNeeded(this);
```

Comment:
You could make a Locator that would match either button.
```suggestion
        protected final WebElement saveButton = Locator.union(Locator.button("Save"),
                Locator.buttonContainingText("Finish")).findWhenNeeded(this);
```

Reply:
Awesome, thanks @labkey-tchad. I don't think I've ever used the Locator.union. I'll give it a try.